pxteditor/events.ts

```diff
@@ -0,0 +1,20 @@
+namespace pxt.events {
+
+    export type EditorType = 'blocks' | 'ts';
+
+    export interface PXTEvent {
```

Comment:
don't use the product name here. we're already under the pxt namespace. ``HostEvent``?

Reply:
Sorted.